### ISSUE TITLE
Colon/semicolon next to em-dash?

### ISSUE THREAD

**Issue by tuskpot** [tuskpot/george-eliot_silas-marner#3]
Should the colon/semicolon be removed in these cases?

> …on the other side of the churchyard:﻿—a village which showed…
>…his way of life:﻿—he invited no comer…

**Comment by acabal** [comment]
No, these are OK. This is quite common in almost all older texts. While it's fallen out of favor in modern writing, they haven't been replaced with an equivalent. (Like the old comma-emdash, whose modern equivalent of a long pause is just a regular emdash without a comma.)
